File: codeup/100000572/A.cpp
#include <cstring>
#include <iostream>
using namespace std;
struct person {
  char name[20];
  int count;
} leader[3] = {"Li", 0, "Zhang", 0, "Fun", 0};
int main() {
  int n;
  char name[20];
  scanf("%d", &n);
  for (int i = 0; i < n; i++) {
    scanf("%s", name);
    for (int i = 0; i < 3; i++)
      if (!strcmp(name, leader[i].name)) leader[i].count++;
  }
  for (int i = 0; i < 3; i++)
    printf("%s:%d\n", leader[i].name, leader[i].count);
  return 0;
}

File: codeup/100000572/A.py
#!/usr/bin/python3
class person:
    name = ""
    count = 0

    def __init__(self, n, c):
        self.name = n
        self.count = c


leader_list = [person("Li", 0), person("Zhang", 0), person("Fun", 0)]
n = int(input())
for i in range(n):
    name_str = input()
    for ent in leader_list:
        if(ent.name == name_str):
            ent.count += 1

for ent in leader_list:
    print("{}:{}".format(ent.name, ent.count))
